tools/src/test/java/org/apache/kafka/tools/GetOffsetShellTest.java

```diff
@@ -338,6 +348,10 @@ private void assertExitCodeIsOne(String... args) {
     }
 
     private List<Row> expectedOffsetsWithInternal() {
+        if (cluster.isKRaftTest()) {
```

Comment:
I prefer to make callers use `expectedTestTopicOffsets` instead of adding if-else here.

Reply:
Sure - I've moved the evaluation of `cluster.isKRaftTest()` to the tests instead of putting it in the helper API